app/src/main/java/org/simple/clinic/overdue/AppointmentRepository.kt

```diff
@@ -116,12 +117,24 @@ class AppointmentRepository @Inject constructor(
   fun overdueAppointments(): Observable<List<OverdueAppointment>> {
     return facilityRepository.currentFacility(userSession)
         .map { it.uuid }
-        .flatMap { facilityUuid ->
-          overdueDao.appointmentsForFacility(
-              facilityUuid = facilityUuid,
-              scheduledStatus = Appointment.Status.SCHEDULED,
-              dateNow = LocalDate.now(UTC)
-          ).toObservable()
+        .withLatestFrom(configProvider.toObservable())
+        .flatMap { (facilityUuid, config) ->
+          val today = LocalDate.now(clock)
+          if (config.highlightHighRiskPatients) {
+            overdueDao.appointmentsForFacility(
+                facilityUuid = facilityUuid,
+                scheduledStatus = Appointment.Status.SCHEDULED,
+                dateNow = today,
```

Comment:
Instead of `dateNow`, do you feel a better name would be `appointmentScheduledBefore`?

Reply:
Totally. Doing this.

Reply:
Actually, I'll name it just `scheduledBefore` because the function anyway returns appointments and there's another param called `scheduledStatus`.